client/src/assets/components/posts/Posts.jsx

```diff
@@ -201,6 +205,25 @@ const Posts = () => {
                         <p>No posts found.</p>
             }
 
+            {/* Pagination Controls */}
+            <div className="flex justify-center mt-6">
+                <button
+                    className="btn btn-primary btn-sm"
+                    onClick={() => setCurrentPage((prev) => Math.max(prev - 1, 1))}
+                    disabled={currentPage === 1}
+                >
+                    Previous
+                </button>
+                <span className="mx-4">Page {currentPage} of {totalPages}</span>
+                <button
+                    className="btn btn-primary btn-sm"
+                    onClick={() => setCurrentPage((prev) => Math.min(prev + 1, totalPages))}
+                    disabled={currentPage === totalPages}
+                >
+                    Next
+                </button>
+            </div>
```

Comment:
I really like the implementation, it works great! Is there anyway you can convert it to the styles from [DaisyUI] (https://daisyui.com/components/pagination/) as mentioned?

Here are the preferred styles:
![image](https://github.com/user-attachments/assets/900fa9b3-40bd-4a4f-9ca5-af498e86d971)
or
![image](https://github.com/user-attachments/assets/f41c4e51-bf34-45d2-8eb1-92993cbc59eb)

Reply:
Also, for a better user experience, hiding the buttons when it is only 1 page would be ideal.